third_party/egeria/deploy.sh

```diff
@@ -3,11 +3,11 @@
 set -e
 set -x
 
-: ${WITHOUT_OPENSHIFT=true}
+: ${WITHOUT_OPENSHIFT=false}
 
 CHART=odpi-egeria-lab
 RELEASE=lab
-NAMESPACE=egeria-catalog
+NAMESPACE=irltest1
```

Comment:
Why was the namespace changed?

Reply:
sorry it was edited by mistake during testing. restored it now. Thanks.